lib/spec/simpleofflinecache.sw-spec.js

```diff
@@ -118,16 +104,17 @@ describe('SimpleOfflineCache instance', function () {
     var request = new Request('/index.html?param=zzz');
 
     var cacheObj = {
-      match: sinon.stub().returns(Promise.resolve(null)),
-      put: sinon.stub().returns(Promise.resolve())
+      match: sinon.stub().returns(Promise.resolve(null))
     };
-    sinon.stub(self.cacheHelper, 'getCache').returns(
+    // Clean the cached request
+    delete offline.cacheRequest;
+    sinon.stub(caches, 'open').returns(
       Promise.resolve(cacheObj));
-    offline.onFetch(request).then(function() {
-      expect(self.cacheHelper.getCache.calledOnce).to.be.true;
+    offline.onFetch(request.clone()).then(function() {
```

Comment:
I had to modify the mock for the cache.match, to return something, if i return null (as i was doing before), we will be doing a fetch, and will be consuming the request body, so comparing them was incorrect.

But you are right, the clone is not necessary there, just modify the `cache.match` mock.